composer.json

```
@@ -36,7 +36,7 @@
         "symfony/event-dispatcher": "^4.4.20||^5.0.11||^6.0",
         "symfony/http-kernel": "^4.4.20||^5.0.11||^6.0",
         "symfony/polyfill-php80": "^1.22",
-        "symfony/psr-http-message-bridge": "^1.2||^2.0",
+        "symfony/psr-http-message-bridge": "^1.2||^2.0||^6.4",
```

```suggestion
        "symfony/psr-http-message-bridge": "^1.2||^2.0||^6.4||^7.0",
```

so that this supports also Symfony 7 (which will be released at the same date than 6.4)

---

Supporting Smyonfy 7 is more involved than this, and we already have a PR open. See https://github.com/getsentry/sentry-symfony/pull/761.
I will focus this PR on 6.4 only.